pinot-common/src/main/java/com/linkedin/pinot/common/segment/fetcher/SegmentFetcher.java

```diff
@@ -20,15 +20,15 @@
 
 import java.io.File;
 
+
 public interface SegmentFetcher {
 
   void init(Configuration configs);
 
   void fetchSegmentToLocal(String uri, File tempFile) throws Exception;
 
   /**
-   * @return a list of config keys whose value should not be logged.
+   * Returns a list of config keys whose value should not be logged.
```

Comment:
what's wrong with the javadoc convention? I prefer to have javadocs in interface classes

Reply:
Me too. This is just putting it as description in javadoc. Will add back the *@return*